web/package.json

```diff
@@ -55,7 +55,6 @@
         "lint:js": "eslint . --ext .js,.ts,.tsx",
         "lint:types": "tsc -p tsconfig.json --noEmit",
         "hooks:i": "npx simple-git-hooks",
-        "build:info": "node ./src/setupEnvInfo.js",
```

Comment:
It seems to me that it is worth removing all references to the deleted script
![image](https://user-images.githubusercontent.com/50149163/218076441-7fb17816-3b93-4ecb-a8cb-1db98b6d4867.png)

Reply:
and the file `setupEnvInfo.js` too